internal/pkg/template/workload.go

```diff
@@ -208,7 +208,6 @@ type HTTPHealthCheckOpts struct {
 
 // NetworkLoadBalancerListener holds configuration that's need for a Network Load Balancer listener.
 type NetworkLoadBalancerListener struct {
-	Port            string
```

Comment:
Should we remove the `Aliases` field as well?

Reply:
Currently there is a go template version of `Aliases` used to determine whether the custom resource for custom domain should be rendered or not. 

It could use a `Condition` instead of go template in order to trigger the custom resource at appropriate time, but it adds a potentially confusing and weird-looking `Condition` block (i.e. something like  `UseAlias: !and [!eq [!Ref Aliases, ""], !Condition HasAssociatedDomain]`) so I refrained from doing so. 

What do you think?